doc/proxy-clients.rst

```diff
@@ -101,12 +101,24 @@ You can also pass some options to the Varnish client:
   which tags to invalidate when sending invalidation requests to the caching
   proxy. Make sure that your :ref:`Varnish configuration <varnish_tagging>`
   corresponds to the header used here;
+* ``tag_mode`` (ban): Makes it possible to invalidate tags
+  using purge when for instance using ``xkey`` instead of ban. Supports three
+  modes: ``ban``, ``purge``, and ``purge_single`` for purge per tag.
 * ``header_length`` (default: 7500): Control the maximum header length when
   invalidating tags. If there are more tags to invalidate than fit into the
   header, the invalidation request is split into several requests;
 * ``default_ban_headers`` (default: []): Map of headers that are set on each
   ban request, merged with the built-in headers.
 
+Example for configuring xkey::
+
+    $options = [
+        'tags_header' => 'key',
```

Comment:
if we follow my suggestion to keep the default X-Cache-Tags for xkey tag invalidation, this line can be removed

Reply:
Format is different, so would rather go for `xkey-purge`